index.bs

```diff
@@ -444,13 +444,16 @@ When this method is invoked, the user agent MUST execute the following algorithm
 8. Initialize |issuedRequests| and |currentlyAvailableAuthenticators| to empty lists.
 
 9. For each authenticator currently available on this platform, add the authenticator to |currentlyAvailableAuthenticators|
-    unless the {{ScopedCredentialOptions/attachment}} member of {{options}} is <a>present</a>. In that case, let |attachment|
-    be {{ScopedCredentialOptions/attachment}}, and add the authenticator to |currentlyAvailableAuthenticators| if its attachment
-    modality matches |attachment|.
+    unless the {{ScopedCredentialOptions/attachment}} member of {{options}} is present or the 
+    {{ScopedCredentialOptions/verificationGesture}} member of {{options}} is set to true. In that case, let |attachment|
+    be {{ScopedCredentialOptions/attachment}} and |verificationGesture| be {{ScopedCredentialOptions/verificationGesture}}. 
+    Add the authenticator to |currentlyAvailableAuthenticators| if its attachment modality matches |attachment| and its 
+    user verification modality is set to true.
```

Comment:
I think the intent is something like: for each authenticator present, the platform detects its |authnAttachment| and |authnVerificationCapability|. Then if |attachment| is not undefined, it weeds out all authenticators for which |authnAttachment| does not match |attachment|. Similarly, if |verificationGesture| is not undefined, it weeds out all authenticators for which |authnVerificationCapability| does not match |verificationGesture|. Perhaps we could reword things this way?